src/legacy/core_plugins/interpreter/public/functions/pie.js

```diff
@@ -26,7 +26,7 @@ export const kibanaPie = () => ({
   type: 'render',
   context: {
     types: [
-      'kibana_table', 'null'
+      'datatable'
```

Comment:
q: why we are removing the type `null` or, on the other side: what was the null context type for?